ttrpc-codegen/src/model.rs

```diff
@@ -8,20 +8,15 @@ use crate::str_lit::StrLit;
 use protobuf_support::lexer::float;
 
 /// Protobox syntax
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
```

Comment:
After This change could cause whole project not compatible with lower rust version(<1.66) anymore.
So I think I should limit the Clippy versions instead of this pr.

Reply:
Oh interesting. Should we add a skip statement?  

What is the typical support for older versions of rust?

Reply:
I have file a PR to lock the rust/clippy version https://github.com/containerd/ttrpc-rust/pull/179